TurkishFilter/sections/specific.txt

```diff
@@ -66,6 +66,15 @@ fullfilmcidayim.com,dizispeed.com,babafullfilmizleme.com###adStarter
 /service/advertchannels?$domain=ozgunkocaeli.com.tr|urfanatik.com|dirilispostasi.com|turkuazgazetesi.net|yenialanya.com|bugunkocaeli.com.tr|cagdaskocaeli.com.tr|mansetaydin.com|enkocaeli.com|kocaelikoz.com|kocaelibarisgazetesi.com|seskocaeli.com
 dirilispostasi.com,turkuazgazetesi.net,yenialanya.com,bugunkocaeli.com.tr,cagdaskocaeli.com.tr,mansetaydin.com,seskocaeli.com,kocaelibarisgazetesi.com,kocaelikoz.com,enkocaeli.com##.block-pano > div.us
 !
+||dafflix.com/olley.mp4
+||dafflix.net/yenipop2.js
+dafflix.net##.player-ustu-metin > a[rel="nofollow"]
+dafflix.net##body > div[data-type="pgskin"]
+dafflix.net##.right.floated > div[data-gets] > a[href][target="_blank"] > img
+||dafflix.net/akk.gif
+||dafflix.net/2rek.webp
+dafflix.net##.rek-wrap > *:not(b)
+dafflix.net##.rek-0
```

Comment:
I don't see
```
dafflix.net##.player-ustu-metin > a[rel="nofollow"]
dafflix.net##body > div[data-type="pgskin"]
||dafflix.net/akk.gif
||dafflix.net/2rek.webp
```
For the rest of the rules, change the domain to `dafflix.*`

Reply:
> I don't see

I know, maybe they'll use it again later. Just restore.

Reply:
@dcxstq2 what about the rules above?

Reply:
Rules should only be deleted if the website is permanently down.

Reply:
I mean that the rules above are not used and we can remove them from the pull request.
We only add necessary rules.

Reply:
removed